src/main/java/uk/ac/ebi/eva/pipeline/model/PopulationStats.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2016 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.model;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.util.Map;
+
+/**
+ * @author Jose Miguel Mut Lopez &lt;jmmut@ebi.ac.uk&gt;
+ */
+public class PopulationStats {
```

Comment:
using @Indexed, @Document and @PersistenceConstructor in next commit.

@DBRef is not what we are using.

In the future we can profile if we should use a @CompoundIndex on vid, cid and sid